### DISCUSSION TITLE
Claim Depth! How Do I Make My Claim Lower?

### DISCUSSION THREAD

**Discussion by TheGorshWithin** [GriefPrevention/GriefPrevention#1811] [Q&A]
Hello! My base resides at around Z -44, and due to this it says my chest are unprotected. Along with that sticky pistons do not work as they should due to the depth claim issue.

**Comment by bigpreshkt** [comment]
Make you've edited your config to set `MaxDepth` to at least `-64` for 1.18, and make sure you are running the latest release candidate - or better consider running  https://github.com/TechFortress/GriefPrevention/commit/0e3a2e979161327946365e71b97bbcf74f572e6b (downloadable [here](https://jitpack.io/com/github/TechFortress/GriefPrevention/0e3a2e9791/GriefPrevention-0e3a2e9791.jar)) instead of 16.18-RC1 due to a fixed bug with automatic claim depth extension.

See also quite a few issues raised on exactly this topic - e.g. [closed issues mentioning "depth"](https://github.com/TechFortress/GriefPrevention/issues?q=is%3Aissue+is%3Aclosed+depth)
